Tests/MoneyConverterTest.php

```diff
@@ -1,19 +0,0 @@
-<?php
```

Comment:
You know this better than me, so trying to learn - what was this and why is it removed?

Reply:
It was there to make currency names in string only. But now due to type hinting, this check has become obsolete. It will throw now TypeError instead of MoneyException